core/tests/test_utils.py

```diff
@@ -164,10 +196,16 @@ def signup_superadmin_user(self):
     def log_line(self, line):
         """Print the line with a prefix that can be identified by the
         script that calls the test.
+
+        Args:
+            line: str. A signal message.
         """
         print '%s%s' % (LOG_LINE_PREFIX, line)
 
     def _delete_all_models(self):
+        """Deletes all keys. Subclasses of TestBase
+        should override this method.
```

Comment:
Not sure why the line break is so early on the previous line. You could move some words up and still remain within the 80 char limit?